### PR TITLE
NMS-15158: Add flow table

### PR DESCRIPTION
**Pull request by emarsh1995**
- Update `operation/deep-dive/flows/introduction.adoc`

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://opennms.atlassian.net/browse/NMS-15158

[NMS-15158]: https://opennms.atlassian.net/browse/NMS-15158?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/flows/introduction.adoc [hunk 1]
```diff
@@ -2,44 +2,88 @@
...
-* Application classification
-* Horizontal scaling
-* Enterprise reporting (generate PDF reports)
-* Top K statistics by interface, application, host, conversation with QoS
+* A platform to collect, persist, and visualize flows, with support for NetFlow versions 5 and 9, IPFIX, and sFlow.
+* Inventory enrichment (mapping to {page-component-title} nodes).
+* Application classification.
+* Horizontal scaling.
+* Enterprise reporting.
+* Top K statistics by interface, application, host, conversation with QoS.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/introduction.adoc:18]:
```suggestion
* Top K statistics by interface, application, host, and conversation with QoS.
```

File: docs/modules/operation/pages/deep-dive/flows/introduction.adoc [hunk 2]
```diff
@@ -2,44 +2,88 @@
...
 * Enriched flows are persisted in Elasticsearch and/or forwarded to Kafka.
-* You can use the OpenNMS streaming analytics tool to aggregate flows and output aggregates to Elasticsearch, Cortex, or Kafka
-* The REST API supports generating both summaries and time series data from the flows or flow aggregates stored in Elasticsearch.
-* Use OpenNMS Helm to visualize flows and/or flow aggregates:
-** Use the "Flow Deep Dive" dashboard with an OpenNMS Flow datasource to visualize flows and flow aggregates stored in Elasticsearch.
-** Use the "Cortex Flow Deep Dive" dashboard with a Prometheus datasource to access flow aggregates stored in Cortex.
+* (Optional) The OpenNMS streaming analytics tool aggregates flows and outputs them to Elasticsearch, Cortex, or Kafka.
+* (Optional) The REST API generates summaries and time series data from the flows or aggregates stored on Elasticsearch.
+* OpenNMS Helm visualizes flows and aggregates:
+** The Flow Deep Dive dashboard visualizes flows and aggregates that are stored in Elasticsearch using a flows datasource.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/introduction.adoc:42]: Is it ES using the flows datasource or the dashboard? If the latter, suggest:

```suggestion
** The Flow Deep Dive dashboard uses a flows datasource to visualize flows and aggregates that are stored in Elasticsearch.
```
I think it's probably the former, but we should reword the sentence to make that clearer. 
Maybe:

The Flow Deep Dive dashboard visualizes flows and aggregates that are stored in Elasticsearch from a flows datasource.

File: docs/modules/operation/pages/deep-dive/flows/introduction.adoc [hunk 3]
```diff
@@ -2,44 +2,88 @@
...
+** The Cortex Flow Deep Dive dashboard visualizes aggregates that are stored in Cortex using a Prometheus datasource.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/flows/introduction.adoc:43]: Same question as line 42. Which is using the datasource?

```suggestion
** The Cortex Flow Deep Dive dashboard uses a Prometheus datasource to visualize aggregates that are stored in Cortex.
```

If the former:

The Cortex Flow Deep Dive dashboard visualizes aggregates that are stored in Cortex from a Prometheus datasource.

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
A few minor changes. Thanks!

File: docs/modules/operation/pages/deep-dive/flows/introduction.adoc (hunk 2 above)
Comment (jeffgdotorg) [reply docs/modules/operation/pages/deep-dive/flows/introduction.adoc:42]: I suggest:
The Flow Deep Dive dashboard visualizes flows and aggregates, retrieving their data from Elasticsearch via a flows datasource.

File: docs/modules/operation/pages/deep-dive/flows/introduction.adoc [hunk 4]
```diff
@@ -2,44 +2,88 @@
...
+** Metadata related to associated nodes (such as IDs and categories) are added to the flows.
 * Enriched flows are persisted in Elasticsearch and/or forwarded to Kafka.
-* You can use the OpenNMS streaming analytics tool to aggregate flows and output aggregates to Elasticsearch, Cortex, or Kafka
-* The REST API supports generating both summaries and time series data from the flows or flow aggregates stored in Elasticsearch.
-* Use OpenNMS Helm to visualize flows and/or flow aggregates:
-** Use the "Flow Deep Dive" dashboard with an OpenNMS Flow datasource to visualize flows and flow aggregates stored in Elasticsearch.
-** Use the "Cortex Flow Deep Dive" dashboard with a Prometheus datasource to access flow aggregates stored in Cortex.
+* (Optional) The OpenNMS streaming analytics tool aggregates flows and outputs them to Elasticsearch, Cortex, or Kafka.
+* (Optional) The REST API generates summaries and time series data from the flows or aggregates stored on Elasticsearch.
+* OpenNMS Helm visualizes flows and aggregates:
```
Comment (emarsh1995) [inline docs/modules/operation/pages/deep-dive/flows/introduction.adoc:41]:
```suggestion
. (Optional) The REST API generates summaries and time series data from the flows or aggregates stored on Elasticsearch.
. OpenNMS Helm visualizes flows and aggregates:
```

File: docs/modules/operation/pages/deep-dive/flows/introduction.adoc (hunk 2 above)
Comment (emarsh1995) [reply docs/modules/operation/pages/deep-dive/flows/introduction.adoc:42]:
```suggestion
** The Flow Deep Dive dashboard visualizes flows and aggregates, retrieving their data from Elasticsearch via a flows datasource.
```

**Review by Bonrob2 — COMMENTED** [review]
LGTM. Thanks!

**Review by mmahacek — APPROVED** [review]

**Pull request merged**